src/js/session_report.js

```diff
@@ -94,6 +96,18 @@ export class SessionReport {
     get iceConnectionsLost() {
         return this._iceConnectionsLost;
     }
+    /**
+     * Tells if the RTCSession has failed ICE connection in talking state.
+     */
+    get iceConnectionsFailed() {
+        return this._iceConnectionsFailed;
+    }
+    /**
+     * Tells if the RTCSession has failed Peer connection in talking state.
```

Comment:
"in talking state." - Does. this only happen in Talking state?

Reply:
This is a good question, and I did additional testing for this, verified that this only happens in talking state. 
Before talking state, the connection state is always 'new'
Only after it is moved to talking state, the connection moving to 'connecting' and then go to failed directly or go to 'connected' then later go to 'failed'

Reply:
Will remove in talking state to make this field more generic.